src/main/java/com/fasterxml/jackson/databind/deser/impl/PropertyValueBuffer.java

```diff
@@ -75,14 +75,32 @@ public class PropertyValueBuffer
      */
     protected Object _idValue;
 
+    /**
+     * [databind#562]: Allow use of `@JsonAnySetter` in `@JsonCreator`
+     *
+     * @since 2.18
+     */
+    protected final SettableAnyProperty _anySetter;
+
+    /**
+     * Flag that indicates whether this buffer has been consumed, meaning
+     * that all properties have been read and assigned.
```

Comment:
This requires some more elaboration... looks like this is specifically related to `any-properties` handling?